app/src/main/java/com/tsu/wordsfactory/repository/data/Database.kt

```diff
@@ -0,0 +1,37 @@
+package com.tsu.midtermexam3.data
+
+import android.content.Context
+import androidx.room.Room
+import androidx.room.RoomDatabase
+import com.tsu.wordsfactory.repository.data.DefinitionModel
+import com.tsu.wordsfactory.repository.data.MeaningModel
+
+
+@androidx.room.Database(entities = [WordModel::class, MeaningModel::class, DefinitionModel::class], version = 3, exportSchema = false)
```

Comment:
You could use only `version = 1` if your project in develop (didn't released to user)